src/components/Footer/index.tsx

```diff
@@ -0,0 +1,105 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import CONFIG from '@/lib/configs';
+import { IconType } from '@/lib/icons';
+import { Flex } from '@/styles/containers';
+import { P } from '@/styles/text';
+import FooterImage from '../../../public/images/footer-image.webp';
+import Icon from '../Icon';
+import * as Styles from './styles';
+
+type SocialLink = {
+  href: string;
+  type: IconType;
+};
+
+const socialLinks: SocialLink[] = [
+  {
+    href: CONFIG.facebook,
+    type: 'facebook',
+  },
+  { href: CONFIG.xTwitter, type: 'x' },
+  {
+    href: CONFIG.linkedin,
+    type: 'linkedin',
+  },
+  { href: CONFIG.instagram, type: 'instagram' },
+];
+
+export default function Footer() {
+  return (
+    <Styles.FooterContainer>
+      <Styles.FooterDiv>
+        <Styles.FooterSection>
```

Comment:
looks like the text is being wrapped into a second line
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/463911fb-373f-4669-800d-f64c752d43bf)

compared to design:
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/43983a7c-9c15-4a14-8011-6e474d945183)

maybe we can expand the width here to make it match?